src/index.ts

```diff
@@ -215,7 +215,7 @@ async function testTypesVersion(
 
 function assertPathIsInDefinitelyTyped(dirPath: string): void {
     const parent = dirname(dirPath);
-    const types = /^v\d+$/.test(basename(dirPath)) ? dirname(parent) : parent;
+    const types = /^v\d[\d\.]*$/.test(basename(dirPath)) ? dirname(parent) : parent;
```

Comment:
The version _must_ start with a digit.

Reply:
This allows a trailing `.`, like `v1.2.`

Reply:
Yup, good call. Simplified the regexp.